acasclient/acasclient.py

```diff
@@ -440,6 +440,25 @@ def add_file_to_lot(self, lot_corp_name, file, file_type, writeup=None):
         """
         return self.add_files_to_lot(lot_corp_name, [{"file": file, "file_type": file_type, "writeup": writeup}])
 
+    def _get_cmpdreg_files_content(self, files):
+        """Get the content of cmpdreg files
+
+        Pass an array of cmpdreg_files to download and fill their content attribute
+
+        Args:
+            files: An array of cmpdreg_files to download in the format
+                "subdir": The subdir of the file (usually lot_corp_name)
+                "url": The url of the file to download
+
+        Returns:
+            The same array of cmpdreg_files with the "content" attribute added
+        """
+        for file in files:
+            resp = self.session.get("{}/cmpdreg/MultipleFilePicker/{}".format(self.url, file["url"]))
+            resp.raise_for_status()
+            file["content"] = resp.content
```

Comment:
Sanity check: If the user already has `content` as a key in the dictionary then we want to override it? 

If we don't want to do that, then doing validation check and raising an exception might be a good idea.

Reply:
Also instead of working on the actual input object, do we want to create a copy of it, update that and return that? I am just considering if users would find it weird that their input object got updated.

Reply:
I think this is the one suggestion I won't take. I think if you pass into this function, you would expect it to override your content. I imagine a scenario where you may want to refresh the files objects from whats on server.  I also think its ok to update the object directly.  Not that memory is a problem here but I see this function as more of a utility function to fill out the content immediately after fetching files.  I could have also added this function to be called as an optional `fetch_file_content` parameter when fetching a metalot but I chose to keep that out for now.